installation/scripts/installer.sh

```diff
@@ -88,11 +88,13 @@ if [ ${ADMIN_PASSWORD} ]; then
 fi
 
 USED_DRIVER=$(minikube profile list -o json | jq -r ".valid[0].Config.Driver")
+
 if [[ $USED_DRIVER == "docker" ]]; then
-  MINIKUBE_IP=$(minikube ssh egrep "minikube$" /etc/hosts | cut -f1)
+  MINIKUBE_IP=$(minikube ssh cat /etc/hosts | grep "minikube" | head -n 1 | cut -f1)
```

Comment:
Why you need to cat the file then pipe to grep, when you can do jus `grep "minikube" /etc/hosts | head -n 1`

Also, keep in mind that this same command is used in the kyma-cli. If you think that it is wrong, consider changing it in the kyma-cli as well.

Personally i don't see any difference between those commands (before and after).